great_expectations/profiler/parameter_builder/multi_batch_parameter_builder.py

```diff
@@ -1,15 +1,13 @@
 from abc import ABC
-from typing import List, Optional
+from typing import Dict, List, Optional
 
 import great_expectations.exceptions as ge_exceptions
 from great_expectations import DataContext
 from great_expectations.core.batch import BatchRequest
-from great_expectations.profiler.domain_builder.domain import Domain
-from great_expectations.profiler.domain_builder.util import get_batch_ids
 from great_expectations.profiler.parameter_builder.parameter_builder import (
     ParameterBuilder,
 )
-from great_expectations.validator.validator import Validator
+from great_expectations.profiler.util import get_batch_ids
```

Comment:
Is this import needed?

Reply:
@anthonyburdi This import is used on line 39.

Reply:
My mistake! I did not expand the un-edited code.